packages/noya-state/src/reducers/gradientReducer.ts

```diff
@@ -98,6 +103,15 @@ export function gradientReducer(
         draft.stops.splice(index, 1);
       });
     }
+    case 'setFillGradientStops':
```

Comment:
We probably want to set the entire gradient object here, instead of just the stops. Otherwise there's no way to use the other fields of the gradient asset, like `from` and `to`, which are also important.

I think the behavior you have now is good too though, e.g. if you want to use the same stops for a linear vs. radial gradient. Maybe we can expose both at some point.

Reply:
That's why I wasn't sure about showing the same gradient themes for the linear, radial or angular

Reply:
This might already be clear but just in case, we should still show all gradient themes regardless of the current gradient type:

![Screen Shot 2021-06-21 at 2 02 19 PM](https://user-images.githubusercontent.com/1198882/122827517-4c7cab00-d299-11eb-9a57-fa42772e532b.png)

Picking a gradient asset will switch the gradient type if needed

Reply:
Ohhhhh okey I understand now